tools/modules/AksEdgeDeploy/AksEdgeDeploy-AEC.ps1

```diff
@@ -290,48 +287,6 @@ function Test-ArcEdgeAzModules {
         Install-PackageProvider -Name NuGet -MinimumVersion 2.8.5.201 -Force -Confirm:$false
     } else { Write-Host "NuGet found" -ForegroundColor Green }
 
-    Write-Host "Checking Helm"

```

Comment:
Why are we removing helm from the script?

Reply:
its installed by the Az ConnectedKubernetes module